ovsInit.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#Start OVS in the Contiv container
+
+set -euo pipefail
+
+if [[ $(lsmod | cut -d" " -f1 | grep -q openvswitch) -eq 1 ]]; then
+	modprobe openvswitch
+fi
+
+mkdir -p /var/run/openvswitch
+mkdir -p /var/contiv/log/
```

Comment:
Didn't we switch this to /var/log/contiv at some point?

Reply:
I've copied this from contiv/netplugin. I'll check there as well.

Reply:
we should move to `/var/log/contiv` instead, v2plugin put log in this dir, but k8s not yet, but /var/log is a more common location for log